hail/src/main/scala/is/hail/backend/service/ServiceBackend.scala

```diff
@@ -547,7 +430,6 @@ class ServiceBackendSocketAPI2(
   private[this] val PARSE_VCF_METADATA = 8
   private[this] val INDEX_BGEN = 9
   private[this] val IMPORT_FAM = 10
-  private[this] val GOODBYE = 254
```

Comment:
Why is this removed?

Reply:
It's not used. It was used ages ago when the query-service was an actual service that opened a TCP connection, issued several commands and then closed the connection cleanly by saying GOODBYE.